src/data/full-quiz.ts

```diff
@@ -88,7 +88,7 @@ const fullQuiz = [
   },
   {
     Question:
-      "In a switch statement, the __________________ case executes if none of the options are true.",
+      "In JavaScript, in a switch statement, the __________________ case executes if none of the options are true.",
```

Comment:
I think the wording can be clean up a bit

Proposed fix:
In a JavaScript switch statement, the __________________ case executes if none of the options are true.